spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/types/ColumnOrValue.scala

```diff
@@ -26,7 +27,7 @@ trait ColumnOrValue[T] {
   def column: Column
   def isColumn: Boolean
   def isValue: Boolean
-  def getColumnName: Option[String]
+  def columnNames: Set[String]
```

Comment:
```suggestion
  def getColumnNames: Set[String]
```
Current method name does not fit to pattern.